hubblestack/extmods/returners/splunk_nebula_return.py

```diff
@@ -39,6 +39,8 @@
               - product_group
 '''
 import socket
+# Import AWS details
+from aws_details import *
```

Comment:
Can we change this to `from aws_details import get_aws_details` please? I don't like `*` imports, it's hard to tell where functions are coming from, and can trample your global context.

Same with the other files, but I'll only leave the one comment.